cmd/server/assets/apikeys.html

```diff
@@ -52,30 +57,36 @@ <h2>API keys</h2>
             <th scope="col">Key</th>
             <th scope="col">Type</th>
             <th scope="col">Enabled</th>
+            <th scope="col">Issued (1d)</th>
+            <th scope="col">Issued (7d)</th>
+            <th scope="col">Issued (30d)</th>
           </tr>
         </thead>
         <tbody>
-        {{range .apps}}
+          {{range .apps}}
           <tr>
-            <td>{{.Name}}</td>
```

Comment:
I think some of this got lost in a rebase?

Reply:
It isn't lost so much as reformatted. Though I see there are some other changes (`scope="row" `seems to have been dropped)